dispatch/tests/test_api_articles.py

```diff
@@ -476,3 +476,23 @@ def test_section_name_query(self):
         self.assertEqual(data['results'][0]['headline'], 'Article 2')
         self.assertEqual(data['results'][1]['headline'], 'Article 1')
         self.assertEqual(data['count'], 2)
+
+    def test_author_name_query(self):
+        """Should be able to search for articles by author name"""
+
+        article_1 = DispatchTestHelpers.create_article(self.client, headline='Article 1', slug='article-1')
+        article_2 = DispatchTestHelpers.create_article(self.client, headline='Article 2', slug='article-2')
+
+        person = Person.objects.create(full_name='Test Person2')
```

Comment:
This line is a bit redundant -- I think you can just pass the `full_name` to `DispatchTestHelpers.create_article` and it will handle the person creation.

```py
article_3 = DispatchTestHelpers.create_article(self.client, headline='Article 3', slug='article-3', author='Test Person2')
```